apps/tangle-dapp/app/restake/OverviewCard/ActionButton.tsx

```diff
@@ -11,6 +11,7 @@ import {
   Typography,
   useCheckMobile,
 } from '@webb-tools/webb-ui-components';
+import type BN from 'bn.js';
```

Comment:
import from `@polkadot/utils` to  ensure there are no conflicts between the PolkadotJs API and the type `BN`